content/ar/docs/setup/production-environment/_index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "بيئات الإنتاج"
+description: تصميم عنقود كوبيرنيتيس جاهز للإنتاج
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+يتطلب إنشاء عنقود كوبيرنيتيس بجودة إنتاجية تخطيطًا وإعدادًا دقيقين.
+إذا كان عنقود كوبيرنيتيس الخاص بك سيقوم بتشغيل أحمال عمل حساسة،
+فيجب تكوينه ليكون متينًا وقادرًا على التكَيُّف والصمود.
+تشرح هذه الصفحة الإرشادات لإعداد عنقود جاهز للإنتاج،
+أو لترقية عنقود حالي للاستخدام الإنتاجي.
+إذا كنت على دراية مسبقة بالإعداد الإنتاجي وترغب في الوصول إلى روابط الموارد مباشرةً، انتقل إلى [الخطوات التالية](#what-s-next).
+
+<!-- body -->
+
+## معايير البيئة الإنتاجية
```

Comment:
I'm not sure on this one. I think that معايير is more meaningful here. I am taking the liberty here of interpreting the whole section then translating it, rather than doing it literally.